File: DataVB/Maps/AuthorMap.vb
Imports Dapper.FluentMap.Dommel.Mapping
Imports EntityModels

Namespace Maps

    Public Class AuthorMap
        Inherits DommelEntityMap(Of AuthorEM)

        Public Sub New()
            ToTable("Author")
            Map(Function(a) a.Id).IsIdentity()
            Map(Function(a) a.Books).Ignore()
        End Sub

    End Class

End Namespace

File: AWSLambda/Maps/AuthorMap.cs
﻿using Dapper.FluentMap.Dommel.Mapping;
using EntityModels;

namespace AWSLambda.Maps
{
    public class AuthorMap : DommelEntityMap<AuthorEM>
    {
        public AuthorMap()
        {
            ToTable("Author");
            Map(a => a.Id).IsIdentity();
            Map(a => a.Books).Ignore();
        }
    }
}
